peachjam/views/bills.py

```diff
@@ -28,7 +28,8 @@ def get_context_data(self, **kwargs):
         context["doc_table_show_court"] = False
         context["doc_table_show_jurisdiction"] = False
 
-        del context["facet_data"]["natures"]
+        if "nature" in context["facet_data"]:
```

Comment:
Let's just remove this completely -- IIRC it was only there so that "bill" wasn't the only one in the list, which is now fixed elsewhere.